common/export-common-cartridge-util/src/java/org/sakaiproject/exporter/util/ExportUtil.java

```diff
@@ -64,6 +64,15 @@ String getFilePath(String contentResourceId, boolean encode) {
 		}
 	}
 
+	String getFileName(ContentResource contentResource, boolean encode) {
```

Comment:
Do you need the `encode` parameter as it's always false?

Reply:
I'll remove it.